repos/spack_repo/builtin/packages/py_metatomic_torch/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyMetatomicTorch(PythonPackage):
+    """Torchscript bindings for metatomic"""
+
+    homepage = "https://docs.metatensor.org/metatomic"
+    pypi = "metatomic-torch/metatomic_torch-0.1.2.tar.gz"
+
+    import_modules = ["metatomic.torch"]
+
+    maintainers("HaoZeke", "luthaf", "rmeli")
+    license("BSD-3-Clause", checked_by="HaoZeke")
+
+    version("0.1.4", sha256="c593bbc0fa3a410bd19d4a4a8d0008d5bd1c31a9faaca85b9d6b655ee1133bde")
+
+    depends_on("python@3.9:", type=("build", "run"))
+    depends_on("py-vesin", type=("build", "run"))
+    depends_on("py-torch@2.1:", type=("build", "run"))
+    depends_on("py-metatensor-torch@0.8", type=("build", "run"))
+    # >=0.3.0 and <0.4.0
+    depends_on("py-metatensor-operations@0.3", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-metatensor-operations@0.3.0:0.4", type=("build", "run"))
```

Reply:
This includes `0.4.` Do you mean `0.3.0:0.3` (in which case I think `0.3` is roughly equivalent since it matches `0.3` and all `0.3.x` versions)?
```suggestion
    depends_on("py-metatensor-operations@0.3.0:0.3", type=("build", "run"))
```

Reply:
yes, I meant something like `0.3.0:0.3`, which might be worth using for consistency sake with the other recipes (there is no reason to treat a lower bound of `0.3.0` differently from `0.3.2` IMO).

Reply:
Yes, I like `0.3.0:0.3` because the intent is very clear (and I think `@0.3` might also match alpha/beta releases etc, but would need to double check...)